src/vmm/src/guest_config/mod.rs

```diff
@@ -3,3 +3,11 @@
 
```

Comment:
Why did you drop the `RegisterModifier` struct? Shouldn't it be somewhere here?

Reply:
It will be. But in the context of the PR, this is only after the configuration has been created. The modifiers are relevant as a part of the template to create the configuration. This PR is just to focusing on the structure of the configuration.